uaclient/entitlements/tests/test_fips.py

```diff
@@ -177,7 +180,7 @@ def test_enable_errors_on_repo_pin_but_invalid_origin(
 
         assert 0 == m_add_apt.call_count
         assert 0 == m_add_pinning.call_count
-        assert 'ERROR    Cannot setup apt pin' in caplog_text()
+        assert 'Cannot setup apt pin' in caplog_text()
```

Comment:
I think we're actually relaxing the test here; this would no longer catch if we started logging at a higher log level (e.g. CRITICAL, or accidentally logging at an undefined level > logging.ERROR).

I don't think this is that important for this test (because the difference between ERROR and CRITICAL isn't that huge given how we use the logging levels), but for tests where we are currently ensuring we emit WARNING I think it probably is an issue worth addressing (because changing from WARNING to ERROR is a substantial difference in how we would expect users to respond).

(If this change was solely to address the log message matching errors, perhaps we can just drop it?)

Reply:
Dan, yes let's drop for now, I  wanted to get someone's eyes on the approach in case we wanted to construct something that filters caplogs to a specific level. There might be something separate we can iterate on if the need arises, but your fix to tip settles the original issue of CI falling over.

Reply:
Just pushed a changeset that is less invasive for subp by creating a wrapper around _subp